deepchecks/nlp/utils/text_properties.py

```diff
@@ -448,6 +449,19 @@ def average_sentence_length(raw_text: Sequence[str]) -> List[float]:
     return result
 
 
+def count_unique_urls(raw_text: Sequence[str]) -> List[str]:
+    """Return a list of integers denoting the number of unique URLS per text sample."""
+    url_pattern = r'https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+'
```

Comment:
the allow both http and https

Reply:
There is '?' after the character 's' which makes `s` an optional one, so it will match with **http://** and **https://**